vinitD/javascript/assignment2/js2.1.html

```diff
@@ -0,0 +1,111 @@
+                        document.getElementById("pid1").innerHTML = +sec;
+                    }
+                } else if (c > a && c > b && c > d && c > e) {
+                    max = c;
+                    document.getElementById("pid").innerHTML = +max;
+                    if (a > b && a > d && a > e) {
+                        sec = a;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (b > a && b > d && b > e) {
+                        sec = b;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (d > a && d > b && d > e) {
+                        sec = d;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (e > a && e > b && e > d) {
+                        sec = e;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    }
+                } else if (d > a && d > b && d > c && d > e) {
+                    max = d;
+                    document.getElementById("pid").innerHTML = +max;
+                    if (a > b && a > c && a > e) {
+                        sec = a;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (b > a && b > c && b > e) {
+                        sec = b;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (c > a && c > b && c > e) {
+                        sec = c;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (e > a && e > b && e > c) {
+                        sec = e;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    }
+                } else {
+                    max = e;
+                    document.getElementById("pid").innerHTML = +max;
+                    if (a > b && a > c && a > d) {
+                        sec = a;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (b > a && b > c && b > d) {
+                        sec = b;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (c > a && c > b && c > d) {
+                        sec = c;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    } else if (d > a && d > b && d > c) {
+                        sec = d;
+                        document.getElementById("pid1").innerHTML = +sec;
+                    }
+                }
+            }
+        </script>
+    </head>
+    <body>
+        <input type="text" id="num1" />
+        <input type="text" id="num2" />
+        <input type="text" id="num3" />
+        <input type="text" id="num4" />
+        <input type="text" id="num5" />
+        <button onclick="lar()">DO</button>
+        <p id="pid"></p>
+        <p id="pid1"></p>
```

Comment:
why this second tag is needed

Reply:
sir it shows both which is largest and which is second largest